compass/ocean/tests/hurricane/analysis/__init__.py

```diff
@@ -0,0 +1,274 @@
+from compass.step import Step
+
+import netCDF4
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+import matplotlib.gridspec as gridspec
+import numpy as np
+import datetime
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+from importlib import resources
+from scipy import spatial
+import json
+import os
+plt.switch_backend('agg')
+
+
+class Analysis(Step):
+    """
+    A step for producing ssh validation plots at observation stations
+
+    Attributes
+    ----------
+    frmt : str
+        Format for datetimes
+
+    min_date : str
+        Beginning of time period to plot in frmt format
+
+    max_data : str
+        End of time period to plot in frmt format
+
+    pointstats_file : dict
+        Dictionary of pointwiseStats outputs to plot. Dictionary key
+        becomes the lable in the legend.
+
+    observation : dict
+        Dictionary of stations belonging to a certain data product
+    """
+    def __init__(self, test_case, storm):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.hurricane.forward.Forward
+            The test case this step belongs to
+
+        storm : str
+            The name of the storm to be plotted
+        """
+        super().__init__(test_case=test_case, name='analysis')
+
+        self.add_input_file(filename='pointwiseStats.nc',
+                            target='../forward/pointwiseStats.nc')
+
+        self.frmt = '%Y %m %d %H %M'
+        self.storm = storm
+
+    def setup(self):
+        """
+        Setup test case and download data
+        """
+        package = self.__module__
+
+        if self.storm == 'sandy':
+            self.min_date = '2012 10 24 00 00'
+            self.max_date = '2012 11 04 00 00'
+            self.pointstats_file = {'MPAS-O': './pointwiseStats.nc'}
+
+            path = resources.path(package, f'{self.storm}_stations.json')
+            with open(path) as stations_file:
+                self.observations = json.load(stations_file)
```

Comment:
I'm seeing errors with what you have here, not completely sure why.  Maybe try:
```suggestion
            with resources.open_text(package, f'{self.storm}_stations.json') as stations_file:
                self.observations = json.load(stations_file)
```